master_server.py

```diff
@@ -47,13 +54,21 @@ def main(args):
     """Master server main function."""
     servers, has_ui = create_servers(silent=args.silent,
                                      debug_mode=args.debug_mode)
+    broadcast_queues = {server.__class__.__name__: queue.Queue() for server in servers}
     threads = [
-        threading.Thread(target=server.serve_forever)
+        threading.Thread(target=server.serve_forever, name=server.__class__.__name__,
+            args=(broadcast_queues[server.__class__.__name__], ))
         for server in servers
     ]
     for thread in threads:
         thread.start()
 
+    def broadcast_message(text):
+        # Function for easier broadcasting in interactive mode.
+        broadcast = FMP.construct_broadcast(text)
+        for broadcast_segment in broadcast:
+            broadcast_queues[FMP.BASE.cls.__name__].put(broadcast_segment, block=True)
```

Comment:
Can't we iterate over the servers and call their `broadcast_message` method rather than "hijacking" their `broadcast_queues`?